src/vm/threadpoolrequest.h

```diff
@@ -184,10 +184,9 @@ class ManagedPerAppDomainTPCount : public IPerAppDomainTPCount {
     ADID m_id;
     TPIndex m_index;
     DECLSPEC_ALIGN(MAX_CACHE_LINE_SIZE) struct {
-        BYTE m_padding1[64 - sizeof(LONG)];
+        BYTE m_padding1[MAX_CACHE_LINE_SIZE - sizeof(LONG)];
         // Only use with VolatileLoad+VolatileStore+FastInterlockCompareExchange
         LONG m_numRequestsPending;
-        BYTE m_padding2[64];
```

Comment:
I think this second padding is still needed. If I understand its purpose correctly, it was used to ensure that the m_numRequestsPending doesn't share the cacheline with anything even if the ManagedPerAppDomainTPCount was not allocated with alignment (which is actually the case, it is allocated using regular new).